doc/reference_processing.xml

```diff
@@ -2403,6 +2403,58 @@ FROM (SELECT  'POLYGON((0 0, 8 8, 0 16, 0 0))'::geometry geom) As foo;
           </refsection>
     </refentry>
 
+
+    <refentry id="ST_SimplifyPolygonal">
+      <refnamediv>
+        <refname>ST_SimplifyPolygonal</refname>
+        <refpurpose>Computes a boundary-respecting hull of a polygonal geometry, with hull shape determined by a target parameter.</refpurpose>
+      </refnamediv>
+
+      <refsynopsisdiv>
+        <funcsynopsis>
+         <funcprototype>
+            <funcdef>geometry <function>ST_SimplifyPolygonal</function></funcdef>
+            <paramdef><type>geometry </type> <parameter>param_geom</parameter></paramdef>
+            <paramdef><type>float </type> <parameter>vertex_fraction</parameter></paramdef>
+            <paramdef choice="opt"><type>integer </type> <parameter>is_outer = 1</parameter></paramdef>
```

Comment:
sounds like bool really?

Reply:
what is "middle polygon hull"? is it SimplifyVW really?

Reply:
Close, but it will preserve polygonal topology, but SimplifyVW does not do right now.  I guess we could expose it under `SimplifyVW` for the polygonal case instead.